examples/server_profiles.py

```diff
@@ -24,234 +24,216 @@
-
-try:
-    # List available targets
-    print("\nList all available servers and bays for a given enclosure group.")
-    available_targets = oneview_client.server_profiles.get_available_targets(enclosureGroupUri=enclosure_group_uri)
-    pprint(available_targets)
-except HPOneViewException as e:
-    print(e.msg)
-
-
-# Generate a new Server Profile Template based on an existing Server Profile
-new_spt = oneview_client.server_profiles.get_new_profile_template(basic_profile['uri'])
+
+# Transform an server profile
+print("\nTransform an existing profile by supplying a new server hardware type and/or enclosure group.")
+server_transformed = profile.get_transformation(
+    enclosureGroupUri=enclosure_group.data["uri"],
+    serverHardwareTypeUri=hardware_type.data['uri'])
+pprint(server_transformed)
+
+print("Transformation complete. Updating server profile with the new configuration.")
+profile_updated = profile.update(server_transformed)
+pprint(profile_updated.data)
+
+# Create a new Server Profile Template based on an existing Server Profile
+new_spt = profile.get_new_profile_template()
 print('\nNew SPT generated:')
 pprint(new_spt)
 
 new_spt['name'] = 'spt_generated_from_sp'
-
-new_spt = oneview_client.server_profile_templates.create(new_spt)
+new_spt = profile_templates.create(new_spt)
 print('\nNew SPT created successfully.')
 
-oneview_client.server_profile_templates.delete(new_spt)
+new_spt.delete()
 print('\nDropped recently created SPT.')
 
-
 # Delete the created server profile
 print("\nDelete the created server profile")
-oneview_client.server_profiles.delete(basic_profile)
+profile.delete()
 print("The server profile was successfully deleted.")
 
 # Delete the created server profile template
-oneview_client.server_profile_templates.delete(basic_server_template)
+server_template.delete()
+
+# Get profile ports
+print("\nRetrieve the port model associated with a server hardware type and enclosure group")
+profile_ports = server_profiles.get_profile_ports(enclosureGroupUri=enclosure_group.data["uri"],
+                                                  serverHardwareTypeUri=hardware_type.data["uri"])
+pprint(profile_ports)
+
+# Get profile ports
+print("\nRetrieve the port model associated with a server hardware")
+profile_ports = oneview_client.server_profiles.get_profile_ports(
+    serverHardwareUri=server_hardware.data["uri"])
+pprint(profile_ports)
+
+# Get the list of networks and network sets that are available to a server profile along with their respective ports
+print("\nList all Ethernet networks associated with a server hardware type and enclosure group")
+available_networks = server_profiles.get_available_networks(
+    enclosureGroupUri=enclosure_group.data["uri"],
+    serverHardwareTypeUri=hardware_type.data["uri"], view='Ethernet')
+pprint(available_networks)
+
+# Get the all Ethernet networks associated with a server hardware type, enclosure group and scopeuris
+if oneview_client.api_version >= 600:
```

Comment:
This endpoint is supported in all the versions, may I know why this condition is kept here?

Reply:
This may be because of the query parameters, scope_uris param is got added in API version 600.

Reply:
Maybe I should update the comment here.

Reply:
Updated the comment